larray_editor/arrayadapter.py

```diff
@@ -35,29 +35,28 @@ def get_axes_names(self):
         return self.filtered_data.axes.display_names
 
     def get_axes(self):
-        axes = self.filtered_data.axes
-        if len(axes) == 0:
+        if self.filtered_data.size == 0:
             return None
         else:
-            axes_names = axes.display_names
+            axes_names = self.filtered_data.axes.display_names
             if len(axes_names) >= 2:
                 axes_names = axes_names[:-2] + [axes_names[-2] + '\\' + axes_names[-1]]
             return [[axis_name] for axis_name in axes_names]
 
     def get_xlabels(self):
-        axes = self.filtered_data.axes
-        if len(axes) == 0:
+        if self.filtered_data.size == 0:
             return None
-        elif len(axes.labels[-1]) == 0:
+        axes = self.filtered_data.axes
+        if len(axes.labels[-1]) == 0:
```

Comment:
I wonder if this case is possible. In other words, is it possible to have len(axes.labels[-1]) == 0 with self.filtered_data.size > 0 ?